### PR TITLE
chore(deploy): update version in deployment yaml

### PR DESCRIPTION
**Pull request by supereagle**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @lichuan0620 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by lichuan0620** [comment]
blocked by CI check, #32 must be merged first

File: deploy/deploy.yaml [hunk 1]
```diff
@@ -103,7 +103,7 @@
                      "./event_exporter"
                   ],
                   "env": [ ],
-                  "image": "cargo.caicloud.io/sysinfra/event-exporter",
+                  "image": "caicloud/event-exporter:v0.2.0",
```
Comment (lichuan0620) [inline deploy/deploy.yaml:106]: the registry used here should be consistent to the Makefile, which is updated in #32

Comment (supereagle) [reply deploy/deploy.yaml:106]: This deployment is to help users to run this exporter. The images need to be pushed into Docker hub after released.
The latest released version is [v0.2.0](https://hub.docker.com/layers/caicloud/event-exporter/v0.2.0).

**Pull request merged**